apps/src/studio/studio.js

```diff
@@ -978,12 +978,14 @@ function edgeCollidableCollisionDistance (collidable, edgeName, yAxis) {
 function handleActorCollisionsWithCollidableList (
            spriteIndex, xCenter, yCenter, list, autoDisappear)
 {
-  // Traverse the list in reverse order because we may remove elements from the
-  // list while inside the loop:
-  for (var i = list.length - 1; i >= 0; i--) {
+  for (var i = 0; i < list.length; i++) {
     var collidable = list[i];
     var next = collidable.getNextPosition();
 
+    if (collidable.isFading()) {
```

Comment:
This function is used for two types of `Collidable` lists: `Item` and `Projectile`. However, you are now calling two new methods (`isFading` and `beginRemoveElement`) that don't exist on `Collidable` or `Projectile`, but only exist on `Item`.